account_mtd/static/src/js/screen.js

```diff
@@ -34,7 +32,8 @@ odoo.define('account_mtd.screen', function(require) {
 	            	dnt_res = 'true'
 	            }
 	            for(var i=0;i<clientinfo.plugins.length;i++){
-	              plugins.push(encodeURIComponent(clientinfo.plugins[i].name))
+	            	plugins = []
```

Comment:
I dont understand why is plugins set to an empty list inside the loop and is pushing everytime when entering the loop an element?